go.mod

```diff
@@ -1,3 +1,29 @@
 module github.com/polygon-io/client-golang
 
-go 1.13
+go 1.17
+
+require (
+	github.com/gin-gonic/gin v1.7.7
```

Comment:
Did we review the licenses on each of this libraries?  Do we have defined good/bad license types?

Reply:
yeah, they all have permissive licenses (MIT or BSD). not sure we need to worry about this just yet but there are tools out there to check dependency licenses

Reply:
why are we importing gin?

Reply:
we're not, this is an outdated comment